wled00/FX.cpp

```diff
@@ -6643,7 +6643,7 @@ static const char _data_FX_MODE_GRAVIMETER[] PROGMEM = "Gravimeter@Rate of fall,
 //   * JUGGLES      //
 //////////////////////
 uint16_t mode_juggles(void) {                   // Juggles. By Andrew Tuline.
-  if (SEGLEN == 1) return mode_static();
+  //if (SEGLEN == 1) return mode_static();
```

Comment:
Please delete these commented-out checks. No need to keep them in the code - github always has the history.

Reply:
Done!